sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -674,6 +674,15 @@ public virtual async Task CloseAsync(CancellationToken cancellationToken = defau
         ///     </item>
         ///   </list>
         /// </exception>
+        /// <exception cref="ArgumentOutOfRangeException">
```

Comment:
I don't think this exception can be thrown from this overload as it doesn't take the reason or error description parameters.

Reply:
Removed, thanks :)